### ISSUE TITLE
Player doesn’t start any video

### ISSUE THREAD

**Issue by KuerosN** [jellyfin/Swiftfin#419]
### Describe the bug

Hi, whatever video, movie or tv show I try to play on Swiftfin doesn’t start:

![9D5947C1-7771-4A8A-AA73-88132B9BE7A7](https://user-images.githubusercontent.com/104448300/165337496-a96b39f3-35b3-405a-8eab-9cccc5de90b2.png)

The play button is unresponsive, just grey out after tapping, then again white and the screen stays black, same thing for any other control. 

Thank you.

### Application version

1.0.0 (74)

### Where did you install the app from?

TestFlight

### Device information

iPhone 11 Pro

### OS version

iOS 15.4.1

### Jellyfin server version

10.7.7 stable

**Comment by LePips** [comment]
Thank you for the issue. Is this video transcoded or direct play? Can you provide logs from your server when attempting to play this?

**Comment by KuerosN** [comment]
It doesn't appear any log file if I try to start an episode on Swiftfin, but it appears whenever I start from Jellyfin app.

I think it is direct play.

**Comment by LePips** [comment]
Would you still be able to provide the log file? There are a few things that we can look for surrounding the playback event.

**Comment by KuerosN** [comment]
Yeah no problem

[FFmpeg.Transcode-2022-04-26_18-13-41_ddfd90153d9e7ec63705c6edd4a89455_3155e15e.log](https://github.com/jellyfin/Swiftfin/files/8565223/FFmpeg.Transcode-2022-04-26_18-13-41_ddfd90153d9e7ec63705c6edd4a89455_3155e15e.log)

Thank you for helping me out.
m

**Comment by KuerosN** [comment]
I already tried it.
Done it again, nothing happens, just black screen.

**Comment by LePips** [comment]
Bummer. The log you gave me was for the ffmpeg, could you also provide the Jellyfin log?

**Comment by KuerosN** [comment]
Is this one? Sorry if I'm wasting your time :(
[log_20220426.log](https://github.com/jellyfin/Swiftfin/files/8565374/log_20220426.log)

**Comment by LePips** [comment]
Yep that one, thanks! Are you using TLS 1.3? If so, VLCKit doesn't support it. Your behavior mirrors the behavior of being able to use the app like normal but not being able to play media using TLS 1.3.

**Comment by KuerosN** [comment]
Sorry, how do I check if I'm using it and how I can disable it?

**Comment by KuerosN** [comment]
Ok, I think I can stop here, thank you a lot, for now I'll keep using Jellyfin app.

**Comment by martabal** [comment]
I had the exact same issue. I installed Swiftfin a few weeks ago so I just deleted and re downloaded Swiftfin and it solved the issue. Hope this will help

**Comment by KuerosN** [comment]
I tried, nope. Doesn't work.

**Comment by JonathanVusich** [comment]
@LePips I am running an instance of Jellyfin 10.7.7 just on my LAN and have run into this issue for certain movies. I have tried other media and have not any issues. How can we debug these kinds of issues?

**Comment by LePips** [comment]
Sadly, we are unable to really debug instances of media not playing due to VLCKit. Some issues are known like TLS 1.3 not being supported but otherwise we aren't able to find root causes to some issues.

Something that we can control is setting Force Direct Play which fixes some things as the device profiles still need to be correctly configured. However this requires trial and error.

**Comment by antnythr** [comment]
Hi, just wanted to say that I am experiencing the same issue. Nothing plays regardless of whether Force Direct Play is enabled. If I enable Native Player, most movies will play, but any with HEVC will play only audio.

Server is Arch Linux with 10.7.7, upgraded to 10.8 which didn't fix issue.
Host is AppleTV 4K (15.5) and iPhone (15.5) on my local network.

No reverse proxy or external network connections.
Can't really say anything about TLS 1.3 that's been mentioned.

This setup used to play every file I threw at it, but for some reason it has recently (last few weeks) stopped working.

This is what shows up in my log file when attempting to open a file:

`[2022-06-02 23:02:01.172 -05:00] [INF] [74] Jellyfin.Api.Helpers.MediaInfoHelper: User policy for "test". EnablePlaybackRemuxing: True EnableVideoPlaybackTranscoding: True EnableAudioPlaybackTranscoding: True
[2022-06-02 23:02:01.172 -05:00] [INF] [110] Jellyfin.Api.Helpers.MediaInfoHelper: StreamBuilder.BuildVideoItem( Profile="Anonymous Profile", Path="/opt/mediasrv/media/Movies/testfilm.mp4", AudioStreamIndex=null, SubtitleStreamIndex=null ) => ( PlayMethod=DirectPlay, TranscodeReason=0 ) "media:/videos/e2fa04d3-e8bb-7de8-bb3a-6686a74f948f/stream.mov?MediaSourceId=e2fa04d3e8bb7de8bb3a6686a74f948f&Static=true&VideoCodec=hevc&AudioCodec=aac&AudioStreamIndex=1&api_key=<token>&SubtitleMethod=Encode&Tag=beaac0b7535c6331d866953fae9c4d54"`

Any other thoughts on what the issue could be or how to resolve it?

**Comment by KuerosN** [comment]
Goodmorning @antnythr
OP here, earlier this week I set up my reverse proxy using Caddy and this [guide](https://www.reddit.com/r/jellyfin/comments/hotycl/windows_and_caddy_v2_reverse_proxy_guide/).
I had some minor errors but the reverse proxy with dedicated url through NoIP works properly. 

With your reply I remembered to download and try Swiftfin again and, to my great surprise, now it can play any format - between the ones I have on the server - of video without even forcing Direct Play.

Myabe you could try to set up a reverse proxy and see if it will fix the problem.

PS: I'm pretty sure that the SSL certificate makes the difference here.

**Comment by antnythr** [comment]
Excuse my ignorance, but isn't a reverse proxy only for allowing external network access? If I'm not trying to use external network access, than the issue shouldn't be related to not having it set up, right?

If I had to guess at when things stopped working, it was after installing Swiftfin build 75. I know things definitely worked when using build 74. Is there a way to get an older build of Swiftfin other than compiling it myself?

**Comment by KuerosN** [comment]
Sorry for the late reply.
Unfortunately I don't know how to go back to the previous release.
By the way, I suggested you, from my experience, that through a reverse proxy, Swiftfin could work without any problem.

Hopefully you find a solution, have a nice day.
